app/controllers/reviews_controller.rb

```diff
@@ -161,6 +161,11 @@ def vote_down
       current_user.vote_against(@review)
     end
 
+    if ((@review.votes_against > -9) && @review.comment != "")
```

Comment:
Also, is it necessary to check that the review's comment isn't empty. It won't be displayed unless there's text in the comment.